prisma/seed.ts

```diff
@@ -0,0 +1,133 @@
+import { promises } from "node:fs";
+import { z } from "zod";
+import { ItemCategory, ItemRarity, PrismaClient } from "../src/generated/prisma";
+import { heroes } from "../src/lib/constants/heroData";
+import { StatNames } from "../src/lib/constants/stats";
+const { readFile } = promises;
+
+type PrismaTransaction = Parameters<Parameters<typeof prisma.$transaction>[0]>[0];
+
+const prisma = new PrismaClient();
+
+const GuidValue = z.object({
+  GUID: z.string(),
+  Value: z.string(),
+});
+
+const Buff = z.object({
+  GameValueGUID: z.string(),
+  IconGUID: z.string(),
+  Name: z.string(),
+  Description: z.string(),
+  DescriptionFormatted: z.string(),
+  FormattedText: z.string(),
+  Value: z.number(),
+});
+
+const StadiumDataSchema = z.record(
+  z.string(),
+  z.object({
+    GUID: z.string(),
+    Name: z.string(),
+    Description: z.string(),
+    DescriptionFormatted: z.string().nullable(),
+    Cost: z.number().nonnegative(),
+    Hero: GuidValue.nullable(),
+    Rarity: GuidValue,
+    Category: GuidValue,
+    Buffs: z.array(Buff),
+    GameValues: z.array(z.any()),
+  }),
+);
+
+async function createStats(tx: PrismaTransaction) {
+  return await tx.stat.createManyAndReturn({
+    data: StatNames.map((statName) => ({
+      name: statName,
+      iconURL: `images/stats/${encodeURIComponent(statName)}.webp`,
```

Comment:
Would it make sense for this to have a leading `/`?

```suggestion
      iconURL: `/images/stats/${encodeURIComponent(statName)}.webp`,
```

Reply:
Makes sense to me!